src/concrete/ml/pandas/client_engine.py

```diff
@@ -37,7 +41,9 @@ def keygen(self, keys_path: Optional[Union[Path, str]] = None):
         else:
             self.client.keygen(True)
 
-    def encrypt_from_pandas(self, pandas_dataframe: pandas.DataFrame) -> EncryptedDataFrame:
+    def encrypt_from_pandas(
+        self, pandas_dataframe: pandas.DataFrame, schema: Optional[Dict] = None
```

Comment:
a schema is optional. If set, it should follow a specific format. 

if needed, we could also handle the output of `get_schema` (pandas data-frames) as an input here